### DISCUSSION TITLE
Helpdesk progress bar

### DISCUSSION THREAD

**Discussion by kamcio2603** [YetiForceCompany/YetiForceCRM#14958] [How to] [answered]
How can I make visible progress bar on ticket view as on demo CRM?

![image](https://user-images.githubusercontent.com/175831/125431824-c5fc927f-c840-47c1-aec9-1bc613184c35.png)

**Comment by rskrzypczak** [comment, marked as answer]
Here is your answer: https://github.com/YetiForceCompany/YetiForceCRM/issues/12855
